CHANGELOG.rst

```diff
@@ -3,33 +3,48 @@
 0.20.0
 ------
 
-This release includes improvements to the UX on the UI, internal changes to the Knowledge Graph, a bugfix on the core service
-and support for running sessions in a separate namespace.
+Renku ``0.20.0`` tidies up the RenkuLab session start sequence, as well as an assortment of improvements and big fixes. 
 
-Features
-~~~~~~~~
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
 
-* **Renku core service**: add workflow export endpoint
-* **Renku CLI**: inform users about push in renku save command
-* **UI**: improve UX around sessions 💻 (unified start flow, better logs, error handling)
-* **UI & Gateway**: adopt the latest layout on login and logout pages 🎨
-* **Sessions**: support for running sessions in a separate namespace 
-* **Sessions**: caching jupyter servers
+**🌟 New Features**
 
-Bug fixes
-~~~~~~~~~~
+* *None this release*
```

Comment:
I would leave this section out entirely -- if there are no changes, no need to specify it

Reply:
Idk... I feel most of the time there will be either 2 or even 1 section, it would be weird reading `None this release` many times. But I'm ok with leaving it as is if you prefer it

Reply:
Yea, that's a good point... Maybe better not to shoot ourselves in the foot always making it obvious to our users that we aren't releasing new features 😛